lib/FSi/DoctrineExtensions/Mapping/Driver/AbstractAnnotationDriver.php

```diff
@@ -51,27 +53,27 @@ public function getBaseMetadataFactory()
     /**
      * {@inheritdoc}
      */
-    public function loadClassMetadata(ClassMetadataInterface $metadata)
+    public function loadClassMetadata(ClassMetadataInterface $metadata): void
     {
         if ($this->getBaseMetadataFactory()->isTransient($metadata->getClassName())) {
             return;
         }
+
         $this->loadExtendedClassMetadata($this->getBaseMetadataFactory()->getMetadataFor($metadata->getClassName()), $metadata);
     }
 
     /**
```

Comment:
i think we should remove docblocks which duplicate information with functions' signatures